apps/health_campaign_field_worker_app/lib/pages/inventory/reports/report_details.dart

```diff
@@ -55,6 +55,7 @@ class _InventoryReportDetailsPageState
     extends LocalizedState<InventoryReportDetailsPage> {
   static const _productVariantKey = 'productVariant';
   static const _facilityKey = 'facilityKey';
+  bool _isLoading = false;
```

Comment:
@naveen-egov We can remove this unused variables

Reply:
@Ramkrishna-egov  Resolved